spec/features/user_creates_list.spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+feature 'user adds new item', %{
+  As an authenticated user
+  I want to create a list
+  So that so that people can see it
+
+  Acceptance Criteria:
+  - []  I must be logged in
+  - []  I must provide title and date
+  - []  I must recieve an error message for invalid form
+  - []  I must get a success message and reload the page
+} do
+  let!(:user) { FactoryGirl.create(:user) }
+
+  scenario 'user successfully adds product' do
+    sign_in(user)
+    visit users_path
+    fill_in 'Item name', with: 'New item'
+    fill_in 'List', with: 'groceries'
+    click_button 'Create Product'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.